tests/rptest/tests/consumer_offsets_migration_test.py

```diff
@@ -49,26 +49,48 @@ def test_migrating_consume_offsets(self, failures, cpus):
 
         self.redpanda.start()
         self._client = DefaultClient(self.redpanda)
-        # set of failure suppressed nodes - required to make restarts deterministic
-        suppressed = set()
+
+        # Set of nodes that are undergoing some destructive operations like fault injections
+        # or restarts. We only want to do one of those at a time to avoid conflicts.
+        busy_nodes = set()
+        # synchronize access to busy_nodes set.
+        busy_nodes_lock = threading.RLock()
```

Comment:
> Fixes synchronization around failure injection and
node restarts.

could you add a little context into the commit message about what was wrong and how it was fixed?

> Looped through the test 50 times locally and didn't
run into any issues

this is great context to add to the PR cover letter, but doesn't add much value in the commit message.

Reply:
I updated the commit message and did a force push.